src/components/Pipelines/ParametersList/ParametersList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import pipelineParameterMapping from '../../helpers/pipelineParameterMapping';
+
+const ParametersList = ({ parameters }) =>
+  Object.keys(parameters).map(parameterName =>
+    <div key={parameterName}>
+      <i>{pipelineParameterMapping(parameterName)}</i>:{' '}
+      <code>{String(parameters[parameterName])}</code>
```

Comment:
I am not sure I am fond of seeing `true` and `false` here. We have dedicated icon template for this.

Reply:
The thing is that parameters can hold even strings, or other values, it is general system for holding a parameter. For now we use only booleans, but I think we cannot count on that.

Reply:
Very well, let's keep it this way for now.